source/lex.tex

```diff
@@ -319,8 +319,8 @@
 phases 3 through 6. The categories of preprocessing token are: header names,
 placeholder tokens produced by preprocessing \tcode{import} and \tcode{module} directives
 (\grammarterm{import-keyword}, \grammarterm{module-keyword}, and \grammarterm{export-keyword}),
-identifiers, preprocessing numbers, character literals (including user-defined character
-literals), string literals (including user-defined string literals), preprocessing
+identifiers, preprocessing numbers, \grammarterm{character-literal}s (including user-defined character
+literals), \grammarterm{string-literal}s (including user-defined string literals), preprocessing
```

Comment:
This change is incorrect. User-defined {character,string} literals are not *character-literal* or *string-literal* tokens.

Reply:
Reverted.